src/02-parser/grammar.mly

```diff
@@ -454,12 +454,20 @@ mark_position(X):
   x = X
   { {it= x; at= Location.make $startpos $endpos}}
 
+param_with_variance:
+  | EQUAL? p = PARAM
```

Comment:
Following OCaml's syntax, we should drop `EQUAL` and treat parameters without prefix as invariant.